sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonRuleRepositoryTest.java

```diff
@@ -40,7 +40,7 @@ public void createRulesTest() {
 
     List<RulesDefinition.Rule> rules = repository.rules();
     assertThat(rules).isNotNull();
-    assertThat(rules).hasSize(65);
+    assertThat(rules).hasSize(66);
```

Comment:
We should change that test, it makes it cumbersome to implement rules in parallel.

Reply:
I agree! Let's do it in a separate PR!